cmd/scheduler/main.go

```diff
@@ -0,0 +1,25 @@
+package main
+
+import (
+	"log"
+	"net/http"
+
+	"github.com/denisushakov/todo-rest/internal/config"
+	"github.com/denisushakov/todo-rest/pkg/router"
+)
+
+func main() {
+	cfg := config.MustLoad()
```

Comment:
Зачем такая сложность с возвратом пустой структуры? Можно же просто ничего не возвращать. Есть так же функция `init` в Го https://www.8host.com/blog/kak-rabotaet-funkciya-init-v-go/
```suggestion
	config.MustLoad()
```

Reply:
Я просто подумал что у меня в структуре конфигурации что-то будет, а когда понял что пока ничего, решил уже не переделывать и оставить как заглушку